src/hooks/useContract.ts

```diff
@@ -97,16 +119,26 @@ export function useTokenContract(tokenAddress?: string, withSignerIfPossible?: b
   return useContract(tokenAddress, ERC20_ABI, withSignerIfPossible)
 }
 
-
 export function useMulticallContract(): Contract | null {
   const { chainId } = useActiveWeb3React()
-  
+
   return useContract(chainId && MULTICALL_NETWORKS[chainId], MULTICALL_ABI, false)
 }
 
+export const useStartFiToken = (withSignerIfPossible?: boolean): Contract | null => {
+  return useContract(STARTFI_TOKEN_NETWORK[3].address, STARTFI_TOKEN_ABI, withSignerIfPossible)
+}
 
- 
- 
- 
+export const useStartFiNft = (withSignerIfPossible?: boolean): Contract | null => {
+  /*   const { chainId } = useActiveWeb3React()
+  const networkId = chainId ? chainId : '3'
+  console.log('network id', networkId) */
+  return useContract(STARTFI_NFT_NETWORK['3'].address, STARTFI_NFT_ABI, withSignerIfPossible)
+}
 
- 
+export const useStartFiMarketplace = (withSignerIfPossible?: boolean): Contract | null => {
+  return useContract(STARTFI_MARKET_PLACE_NETWORK[3].address, STARTFI_MARKET_PLACE_ABI, withSignerIfPossible)
+}
+export const useStartFiRoyality = (withSignerIfPossible?: boolean): Contract | null => {
+  return useContract(STARTFI_RoyaltyNFT[3].address, STARTFI_RoyaltyNFTI, withSignerIfPossible)
```

Comment:
I can accept making it static for now, but plz, later on, consider making it generic using [chainId] . Network Errors should be handled on front end

Reply:
ah sure